advanced/Scripts/query.sh

```diff
@@ -126,22 +126,20 @@ scanDatabaseTable() {
 
     # Mark domain as having been white-/blacklist matched (global variable)
     wbMatch=true
-    # Loop through each result
+
+    # Print table name
+    if [[ ! -z "${result}" ]]; then
```

Comment:
This check is not needed as we already checked for `-z "${result}"` earlier and returned if it was true. Also, Stickler doesn't like it.